src/Tasks.kt

```diff
@@ -9,5 +9,16 @@ fun IntArray.sumOfEven():Int {
 }
 
 fun IntArray.firstTwoLargestIndexes():IntArray {
-    return intArrayOf(-2,-2)
+    if(this.count() < 2 ) return intArrayOf(-1,-1)
+
+    var max = Int.MIN_VALUE
+    var theOtherOne = Int.MIN_VALUE
+
+    this.forEach {
+        if(it > max){
```

Comment:
What about this test array:

[10, 5, 1]

Does your solution work?

Reply:
No, it doesn't